yaci-indexer/src/main/java/org/cardanofoundation/rosetta/yaciindexer/service/AccountServiceImpl.java

```
@@ -0,0 +1,78 @@
+import org.cardanofoundation.rosetta.yaciindexer.domain.model.StakeAccountRewardInfo;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.stereotype.Service;
+import reactor.core.publisher.Mono;
+
+import java.math.BigInteger;
+import java.time.Duration;
+import java.util.Optional;
+import java.util.Set;
+
+@Service
+@Slf4j
+@ConditionalOnExpression("'${store.cardano.n2c-node-socket-path:}' != '' || '${store.cardano.n2c-host:}' != ''")
+public class AccountServiceImpl implements AccountService {
+
+    private final EraStorage eraStorage;
+    private final LocalClientProviderManager localClientProviderManager;
+
+    public AccountServiceImpl(@Nullable LocalClientProviderManager localClientProviderManager,
+                              EraStorage eraStorage) {
+        this.eraStorage = eraStorage;
+        this.localClientProviderManager = localClientProviderManager;
+    }
+
+    @Override
+    public Optional<StakeAccountRewardInfo> getAccountInfo(String stakeAddress) {
+        if (localClientProviderManager == null) {
+            throw new IllegalStateException("LocalClientProvider is not initialized. Please check n2c configuration.");
+        }
+
+        Optional<LocalClientProvider> localClientProvider = localClientProviderManager.getLocalClientProvider();
+        try {
+            var localStateQueryClient = localClientProvider.map(LocalClientProvider::getLocalStateQueryClient).orElse(null);
+
+            if (localStateQueryClient == null) {
+                log.info("LocalStateQueryClient is not initialized. Please check if n2c-node-socket-path or n2c-host is configured properly.");
+
+                return Optional.empty();
+            }
+
+            Address address = new Address(stakeAddress);
+
+            try {
+                localStateQueryClient.release().block(Duration.ofSeconds(5));
+            } catch (Exception e) {
+                //Ignore the error
```

Is there a reason why we can ignore this error?